### DISCUSSION TITLE
C++ 禁忌黑魔法：STMP （上） · ykiko's blog

### DISCUSSION THREAD

**Discussion by giscus** [16bit-ykiko/about-me#6] [Announcements]
# C++ 禁忌黑魔法：STMP （上） · ykiko's blog

mainly about computer science and mathematics

https://ykiko.me/zh-cn/articles/646752343/

**Comment by anonymous1937** [comment]
文章透露着一股黑暗圣经的不明意味。

**Comment by Songbingzhi628** [comment]
``auto = []{}'' 这招仅限TU-local，不然会报exposure错误（比如在module unit里），Clang目前没有实现这个报错，但GCC可以看到。
